src/pages/forside/sections/4-personinfo/5-ditt-nav-kontor/apningstider/Apningstider.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import { Publikumsmottak, Aapningstid } from "types/enhetKontaktInfo";
+import { Heading } from "@navikt/ds-react";
+import { FormattedMessage } from "react-intl";
+import Apningstid from "./Apningstid";
+
+interface Props {
+  publikumsmottak: Publikumsmottak[];
+  valgtMottakId: number;
+}
+
+interface Aapningstider {
+  apningstider: Aapningstid[];
+  andreApningstider: boolean;
+}
+
+const FormaterApningstider = (props: Aapningstider) => {
```

Comment:
Denne kunne kanskje blitt skilt ut i en egen fil for ryddighetens skyld, siden det er en egen komponent med egne props. Ser også ut som at indenteringen her er litt off. Har du satt opp prettier hos deg?

Reply:
Tar dette ved overgang til module.scss